src/backend/axiosconfig.jsx

```
@@ -0,0 +1,213 @@
+    }
+  } catch (error) {
+    console.error(error);
+    onFail({ data: null, msg: "Network Error", status: "error" });
+  }
+};
+
+// GET with token
+export const GET_WITH_TOKEN = async (route, onSuccess = () => {}, onError = () => {}, onFail = () => {
+  Toast("Something went wrong. Please try again.");
+}) => {
+  const token = getToken();
```

ESLint

[error] 'getToken' is not defined.

(no-undef)